api/lib/domain/services/authentication/oidc-authentication-service.js

```diff
@@ -374,3 +364,21 @@ class OidcAuthenticationService {
 }
 
 export { OidcAuthenticationService };
+
+function _monitorOidcError({ data, error, event, message }) {
+  const monitoringData = {
+    message,
+    context: 'oidc',
+    data,
+    event,
+    team: 'acces',
+  };
+
+  if (error) {
+    monitoringData.error = { name: error.name ?? error.constructor.name };
```

Comment:
Ma formulation n'était pas claire du tout, alors je reformule : quelle est l'utilité d'utiliser l'opérateur `??` puisque la propriété `error.name` existe pour toutes les `Error` et que cette propriété ne vaudra jamais `null` ou `undefined` ? Ainsi je suggère plutôt : 

```suggestion
    monitoringData.error = { name: error.constructor.name };
```

Reply:
Je vois la subtitlité et tu as raison on aura toujours `Error` comme valeur et non le bon nom d'erreur attendu. Je fais le changement.